tests/suites/hup/suite.js

```diff
@@ -27,10 +27,13 @@ const fetchOS = async (that, version) => { // {{{2
   }
   const path = join(`/data/images`, `balenaOs-inprogress-${version}.img`)
 
+  // Removing all balenaOS images that don't include the substring inprogress
```

Comment:
if we use the `balena-image-manager` module like this: https://github.com/balena-io/balena-cli/blob/master/lib/utils/cloud.ts#L142-L143 we get caching for free, from what I can tell